MdsSupportingClasses/Collivery.php

```diff
@@ -358,22 +398,22 @@ public function getSuburbs($town_id)
             return $this->cache->get('collivery.suburbs.'.$town_id);
         } else {
             try {
-                $result = $this->client()->get_suburbs($town_id, $this->token);
+                $result = $this->consumeAPI("https://api.collivery.co.za/v3/suburbs", array("town_id" => $town_id), 'GET');
             } catch (SoapFault $e) {
```

Comment:
Please remove all references to catching `SoapFault` or `SoapConnectionException` these are never thrown by `curl` or thrown manually in `consumeAPI()` (as it currently stands)

Reply:
Removed.